### PR TITLE
indicate a successful removal of a machine

### LINKED ISSUE #808: machine rm doesn't output
Treat this as a UX bug. As a user I expect `docker rm` to output something when the command finishes since it takes some time (probably does something important) and just quits silently.

```
$ docker-machine.exe rm docker-vb
# just exits, writes nothing to output
```

the `--debug` flag on the other hand provides _some info_ which mostly specific to the driver of the machine. 

See the following two --debug outputs from a rm executed on virtualbox and azure. As a user at the end of the debug I'd probably love to see somebody telling me `Operation completed successfully` because the current output is just vague and hard to parse for a user.

I think we should have a common `cmdRm` output for all drivers saying "hey we just removed you're good" 

### Virtualbox

```
$ docker-machine.exe --debug rm docker-vb-2
←[34mDEBU←[0m[0000] executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe showvminfo docker-vb-2 --machineread
←[34mDEBU←[0m[0000] STDOUT: name="docker-vb-2"
groups="/"
ostype="Linux 2.6 / 3.x (64 bit)"
UUID="db064914-7359-41ac-bbc1-1f0d1b09e946"
CfgFile="C:\\Users\\ahmetb\\.docker\\machine\\machines\\docker-vb-2\\docker-vb-2\\docker-vb-2.vbox"
SnapFldr="C:\\Users\\ahmetb\\.docker\\machine\\machines\\docker-vb-2\\docker-vb-2\\Snapshots"
LogFldr="C:\\Users\\ahmetb\\.docker\\machine\\machines\\docker-vb-2\\docker-vb-2\\Logs"
hardwareuuid="db064914-7359-41ac-bbc1-1f0d1b09e946"
memory=1024
pagefusion="off"
vram=8
cpuexecutioncap=100
hpet="on"
chipset="piix3"
firmware="BIOS"
cpus=8
pae="on"
longmode="on"
synthcpu="off"
bootmenu="disabled"
boot1="dvd"
boot2="dvd"
boot3="disk"
boot4="none"
acpi="on"
ioapic="on"
biossystemtimeoffset=0
rtcuseutc="on"
hwvirtex="on"
nestedpaging="on"
largepages="on"
vtxvpid="on"
vtxux="on"
VMState="running"
VMStateChangeTime="2015-03-18T20:11:21.868000000"
monitorcount=1
accelerate3d="off"
accelerate2dvideo="off"
teleporterenabled="off"
teleporterport=0
teleporteraddress=""
teleporterpassword=""
tracing-enabled="off"
tracing-allow-vm-access="off"
tracing-config=""
autostart-enabled="off"
autostart-delay=0
defaultfrontend=""
storagecontrollername0="SATA"
storagecontrollertype0="IntelAhci"
storagecontrollerinstance0="0"
storagecontrollermaxportcount0="30"
storagecontrollerportcount0="30"
storagecontrollerbootable0="on"
"SATA-0-0"="C:\Users\ahmetb\.docker\machine\machines\docker-vb-2\boot2docker.iso"
"SATA-ImageUUID-0-0"="7ab6ac28-83bf-4fca-b242-c728c1b13eb8"
"SATA-tempeject"="off"
"SATA-IsEjected"="off"
"SATA-1-0"="C:\Users\ahmetb\.docker\machine\machines\docker-vb-2\disk.vmdk"
"SATA-ImageUUID-1-0"="8883bb6b-f8b0-42ea-82d4-ce0ada3d1637"
"SATA-2-0"="none"
"SATA-3-0"="none"
"SATA-4-0"="none"
"SATA-5-0"="none"
"SATA-6-0"="none"
"SATA-7-0"="none"
"SATA-8-0"="none"
"SATA-9-0"="none"
"SATA-10-0"="none"
"SATA-11-0"="none"
"SATA-12-0"="none"
"SATA-13-0"="none"
"SATA-14-0"="none"
"SATA-15-0"="none"
"SATA-16-0"="none"
"SATA-17-0"="none"
"SATA-18-0"="none"
"SATA-19-0"="none"
"SATA-20-0"="none"
"SATA-21-0"="none"
"SATA-22-0"="none"
"SATA-23-0"="none"
"SATA-24-0"="none"
"SATA-25-0"="none"
"SATA-26-0"="none"
"SATA-27-0"="none"
"SATA-28-0"="none"
"SATA-29-0"="none"
natnet1="nat"
macaddress1="080027A1F72D"
cableconnected1="on"
nic1="nat"
nictype1="virtio"
nicspeed1="0"
mtu="0"
sockSnd="64"
sockRcv="64"
tcpWndSnd="64"
tcpWndRcv="64"
Forwarding(0)="ssh,tcp,127.0.0.1,5043,,22"
hostonlyadapter2="VirtualBox Host-Only Ethernet Adapter #4"
macaddress2="0800279F4C53"
cableconnected2="on"
nic2="hostonly"
nictype2="virtio"
nicspeed2="0"
nic3="none"
nic4="none"
nic5="none"
nic6="none"
nic7="none"
nic8="none"
hidpointing="ps2mouse"
hidkeyboard="ps2kbd"
uart1="off"
uart2="off"
lpt1="off"
lpt2="off"
audio="none"
clipboard="disabled"
draganddrop="disabled"
SessionType="headless"
VideoMode="720,400,0"@0,0
vrde="off"
usb="off"
ehci="off"
VRDEActiveConnection="off"
VRDEClients=0
vcpenabled="off"
vcpscreens=0
vcpfile="C:\Users\ahmetb\.docker\machine\machines\docker-vb-2\docker-vb-2\docker-vb-2.webm"
vcpwidth=1024
vcpheight=768
vcprate=512
vcpfps=25
GuestMemoryBalloon=0
GuestOSType="Linux26_64"
GuestAdditionsRunLevel=1
GuestAdditionsVersion="4.3.20 r96996"
GuestAdditionsFacility_VirtualBox Base Driver=50,1426709498621
GuestAdditionsFacility_Seamless Mode=0,1426709498621
GuestAdditionsFacility_Graphics Mode=0,1426709498620

←[34mDEBU←[0m[0000] STDERR:
←[34mDEBU←[0m[0000] executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe controlvm docker-vb-2 poweroff
←[34mDEBU←[0m[0000] STDOUT:
←[34mDEBU←[0m[0000] STDERR: 0%...10%...20%...30%...40%...50%...60%...70%...80%...90%...100%

←[34mDEBU←[0m[0000] executing: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe unregistervm --delete docker-vb-2
←[34mDEBU←[0m[0000] STDOUT:
←[34mDEBU←[0m[0000] STDERR: 0%...10%...20%...30%...40%...50%...60%...70%...80%...90%...100%
```

What did just happen? :smile: 

### Azure

says removing and removes stuff silently and exits . 

```
$ docker-machine.exe --debug rm ahmetb-azure-machine
←[34mDEBU←[0m[0000] removing ahmetb-azure-machine
``` 

... Probably others are the same.

### PR DESCRIPTION
**Pull request by awsmsrc**
signed off by Luke Roberts luke@speak.io

not an important fix but hey it might save a seal or something

fixes #808

### CODE REVIEW

**Comment by ehazlett** [comment]
Thanks!  Would you mind signing your commits?  https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work  You can use `git rebase -s -i master` to rebase and sign and then `git push -f` to your branch.  This PR will automatically update.

**Comment by awsmsrc** [comment]
i force pushed signed commits

**Comment by ehazlett** [comment]
@awsmsrc you will need to either squash the commits into a single one or sign both commits.  sorry and thanks!

**Comment by awsmsrc** [comment]
@ehazlett did the signing thing

File: commands.go [hunk 1]
```diff
@@ -653,6 +653,8 @@ func cmdRm(c *cli.Context) {
 	}
 	if isError {
 		log.Fatal("There was an error removing a machine. To force remove it, pass the -f option. Warning: this might leave it running on the provider.")
+	} else {
```
Comment (ehazlett) [inline commands.go]: You don't need this `else`.  You can simply do

```
if isError {
...
}

log.Println("Machine removed").
```

...as `log.Fatal` will exit immediately with a non-zero.

**Comment by awsmsrc** [comment]
@ehazlett done

**Comment by ehazlett** [comment]
LGTM
